apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js

```diff
@@ -353,40 +352,42 @@ function updatePointsView(sourceInput) {
 
     const auto_points =
       roundPoints(
-        sourceInput?.name === 'score_auto_percent'
+        sourceInput?.classList?.contains('js-auto-score-value-input-percentage')
           ? (sourceInput?.value * max_auto_points) / 100
-          : form.querySelector('[name=score_auto_points]')?.value,
+          : form.querySelector('.js-auto-score-value-input-points')?.value,
       ) || 0;
```

Comment:
Because the name is no longer always there (see other comment), this query now uses a class instead (which already existed anyway).